### ISSUE TITLE
Using `sf upgrade` attempts to install `Telerik.Sitefinity.All`

### ISSUE THREAD

**Issue by KevinBusch** [Sitefinity/Sitefinity-CLI#83]
Is it possible to have the CLI only upgrade the packages that are installed instead of the entire set of Sitefinity library modules?  We would like to only update `Telerik.Sitefinity.Core` and it's dependencies to keep our instance as thin as possible.  Is this on the roadmap?  Otherwise, we may need to fork this repo just to be able to update our Sitefinity instance.

**Comment by RuzmanovDev** [comment]
Hi, Thanks for the feedback and sorry for the delayed response. Currently we are "installing" Telerik.Sitefinity.All (meaning only copying the packages into the 'packages' folder not installing them into the csproj) in order to gather needed info arround different dependencies. The upgrade tool actually scans your project.config file and in your case will update only Telerik.Sitefinity.Core and it's dependencies. 
The only overhead now is that you will have more nugets into the packages folder. In the future we can think of optimising that.

Long story short: Currently you don't need to fork the repo. Just run the tool :)
